### PR TITLE
Docs propositon

### PR DESCRIPTION
**Pull request by mateuszkp96**
I invite you to discuss :)

### CODE REVIEW

File: docs/LocalDTO.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+        },
+        {
+            "id": 2,
+            "name": "vodka"
+        }
+    ],
+    "website": "www",
+    "contact": {
+        "phoneNumber": "123123123",
+        "email": "mail@gmail.com"
```
Comment (olabudzyn) [inline docs/LocalDTO.md:74]: what about owner id?

File: docs/LocalMapDTO.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Example:
+ ```json
+{
+	"id":"exampleId",
+	"name":"exampleName",
+	"coordinates":{
+		"lat":12.22,
+		"lon":13.33
+	}
+}
```
Comment (olabudzyn) [inline docs/LocalMapDTO.md:11]: do we put somewhere information about the distance between localization of local and client localization? we can display it somewhere

File: docs/LocalDTO.md (hunk 1 above)
Comment (mateuszkp96) [reply docs/LocalDTO.md:74]: IMHO we shouldn't return owner id, its not necessary for frontend purposes

**Comment by mateuszkp96** [comment]
> Which dto correspond to adding a new place? We should have something to add the menu to local and a new product to the menu (menu modification?)

I have not defined all the schemes, all others require designing. Everything above is just an example of how I would see a contract between a frontend team and a backend team.

File: docs/LocalMapDTO.md (hunk 2 above)
Comment (mateuszkp96) [reply docs/LocalMapDTO.md:11]: We should think about whether they should be returned by the same dto

**Pull request merged**